src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Person } from '../../types';
+import { People } from '../People';
+
+type Props = {
+  persons: Person[],
+};
+
+export const PeopleList: React.FC<Props> = ({ persons }) => (
+  <table
+    data-cy="peopleTable"
+    className="table is-striped is-hoverable is-narrow is-fullwidth"
+  >
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
```

Comment:
The names of the columns can be written in an array and re-rendered through the map.